packages/app/provider/index.tsx

```diff
@@ -6,10 +6,34 @@ import { SafeAreaProvider } from './safe-area'
 import { TamaguiProvider } from './tamagui'
 import { UniversalThemeProvider } from './theme'
 import { ToastProvider } from './toast'
-import { WagmiProvider } from './wagmi'
+import { WagmiProvider } from 'wagmi'
+import { RainbowKitProvider } from '@rainbow-me/rainbowkit'
 
 import { ReactQueryDevtools } from '@tanstack/react-query-devtools'
 
+import { baseMainnet, mainnet } from '@my/wagmi'
+import { base as baseMainnetOg, mainnet as mainnetOg } from 'wagmi/chains'
+import { argentWallet, trustWallet, ledgerWallet } from '@rainbow-me/rainbowkit/wallets'
```

Comment:
We need to definitely include the injected wallet.
```suggestion
import {
  argentWallet,
  trustWallet,
  ledgerWallet,
  injectedWallet,
} from '@rainbow-me/rainbowkit/wallets'
```

Reply:
It is included in the default wallet list as a fallback

https://www.rainbowkit.com/docs/custom-wallet-list#built-in-wallets

Reply:
It's not showing up for some reason.

<img width="1334" alt="Screenshot 2024-03-29 at 17 02 58" src="https://github.com/0xsend/sendapp/assets/95193764/b3160a8f-23b9-4c92-ab7e-6c8c23f5e078">

Reply:
What wallet are you using?

Reply:
Rivet.

Reply:
Ah, you probably have 2 wallet extensions running then, since rivet uses EIP 6963

It sees metamask, so it doesn't show the fallback

Good reason for specifically including injected

Reply:
Ah ok, that makes sense. Yeah, when I'm running Rivet, it starts google chrome canary in a brand-new profile, no other extensions installed.